src/meson.build

```diff
@@ -93,7 +93,7 @@ epoxy_has_wgl = build_wgl ? '1' : '0'
 # not needed when building Epoxy; we do want to add them to the generated
 # pkg-config file, for consumers of Epoxy
 gl_reqs = []
-if gl_dep.found()
+if gl_dep.found() and host_system != 'darwin'
```

Comment:
I'm pretty sure this will fail on Windows as well.

The appropriate solution would be to check the type of the `gl_dep` object and adding it only if it's a pkg-config dependency, e.g.

```
if gl_dep.found() and gl_dep.type_name() == 'pkgconfig'
  gl_reqs += 'gl'
endif
```

Same goes for the EGL dependency:

```
if build_egl and egl_dep.found() and egl_dep.type_name() == 'pkgconfig'
  gl_reqs += 'egl'
endif
```